src/goto-programs/remove_unreachable.cpp

```diff
@@ -55,3 +55,9 @@ void remove_unreachable(goto_programt &goto_program)
       it->make_skip();
   }
 }
+
+void remove_unreachable(goto_functionst &goto_functions)
```

Comment:
Could the missing comment header please be added in one of the related pull requests?